aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -300,6 +321,12 @@ public AwsProperties(Map<String, String> properties) {
     this.isS3ChecksumEnabled = PropertyUtil.propertyAsBoolean(properties, S3_CHECKSUM_ENABLED,
         S3_CHECKSUM_ENABLED_DEFAULT);
 
+    this.s3FileIoDeleteBatchSize = PropertyUtil.propertyAsInt(properties, S3FILEIO_DELETE_BATCH_SIZE,
+        S3FILEIO_DELETE_BATCH_SIZE_DEFAULT);
+    Preconditions.checkArgument(s3FileIoDeleteBatchSize > 0 &&
+        s3FileIoDeleteBatchSize <= S3FILEIO_DELETE_MAX_BATCH_SIZE,
```

Comment:
Unfortunately looks like the only decent line break choice for this (both conditions won't fit on the same line).